google_compute_engine/compat.py

```diff
@@ -38,6 +38,10 @@
   import google_compute_engine.distro.debian_8.utils as distro_utils
 elif 'debian' in distro_name and distro_version == '9':
   import google_compute_engine.distro.debian_9.utils as distro_utils
+elif 'suse' in distro_name and distro_version == '11':
+  import google_compute_engine.distro.sles_11.utils as distro_utils
+elif 'suse' in distro_name and distro_version == '12':
+  import google_compute_engine.distro.sles_12.utils as distro_utils
```

Comment:
Probably rename this to suse_wicked

Reply:
I'd actually prefer we leave it as is. We might want to extend the functionality of the distro configs to extend beyond networking.

Reply:
If it extends much beyond networking it might be worth to push version decisions into the distro modules and thus just load a suse, debian, rhel, etc module and then call central entry points for interface_config and whatever else might be needed. Then the complexity of the version decision is pushed to a lower level and it would eliminate what will be an ever increasing if-elif-else block in this file.

Reply:
Ack - we can refactor this as it becomes unwieldy. Currently it seems manageable.